src/qminer/qminer_core.h

```diff
@@ -3219,46 +3219,45 @@ namespace TStreamAggrOut {
 
 	class IInt {
 	public:
-		// retireving value from the aggregate
+		// retrieving value from the aggregate
 		virtual int GetInt() const = 0;
 	};
 
 	class IFlt {
 	public:
-		// retireving value from the aggregate
+		// retrieving value from the aggregate
 		virtual double GetFlt() const = 0;
 	};
 
+	class ISparseVec {
```

Comment:
I recommend removing this -  IValVec<TIntFltKd> already covers it, the vector can be accessed GetValV